src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
```

Comment:
Consider creating a separate package for all classes that extend Task within the Duke package.